### PR TITLE
test(e2e): more tests

### PR DESCRIPTION
**Pull request by omerson**
This PR is related to https://github.com/bleenco/bterm/issues/19

### CODE REVIEW

File: e2e/app.e2e.ts [hunk 1]
```diff
@@ -194,5 +235,110 @@ describe('bterm launch', function() {
...
+      .then(() => this.app.client.click('.maximize'))
+      .then(() => wait(1000))
+      .then(() => this.app.client.browserWindow.getBounds())
+      .then((bounds) => {
+        expect(bounds.width).to.not.equal(width);
+        expect(bounds.height).to.not.equal(height);
+      });
+  });
+
+  it('should set browserWindow.setBounds() correctly', () => {
```
Comment (jkuri) [inline e2e/app.e2e.ts]: this seems to be that you are testing some electron functionallity and not the application?

File: e2e/app.e2e.ts [hunk 2]
```diff
@@ -194,5 +235,110 @@ describe('bterm launch', function() {
...
+        .then((bounds) => {
+          expect(bounds.width).to.equal(50);
+          expect(bounds.height).to.equal(75);
+          expect(bounds.x).to.equal(100);
+          expect(bounds.y).to.equal(200);
+        });
+      });
+  });
+
+  it('should check if loading process passed', () => {
```
Comment (jkuri) [inline e2e/app.e2e.ts]: here too.

File: e2e/app.e2e.ts [hunk 3]
```diff
@@ -194,5 +235,110 @@ describe('bterm launch', function() {
...
+    return this.app.client.waitUntilWindowLoaded()
+      .then(() => this.app.webContents.isLoading())
+      .then((result) => expect(result).to.be.false);
+  });
+
+  it('exposes properties on constructor APIs', () => {
```
Comment (jkuri) [inline e2e/app.e2e.ts]: not sure I understand what exactly do you test here.

File: e2e/app.e2e.ts [hunk 4]
```diff
@@ -84,6 +84,46 @@ describe('bterm launch', function() {
         .then(() => this.app.client.browserWindow.isFocused('.theme-browser > span:nth-child(2)'))
```
Comment (marhi) [inline e2e/app.e2e.ts]: The test will pass if the window has focus, not the second child of the .theme-browser.
I guess this is not the desired outcome.

Comment (omerson) [inline e2e/app.e2e.ts]: You are right. Correct way of checkin this is ` this.app.client.hasFocus('.theme-browser > span:nth-child(2)'))` Thanks.

Comment (omerson) [inline e2e/app.e2e.ts]: PR is updated.

**Pull request merged**